src/db/migrations/20250130113454_add_entries_tenant_id_scope_index.ts

```
@@ -0,0 +1,16 @@
+import type {Knex} from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    return knex.raw(`
+        CREATE INDEX CONCURRENTLY entries_tenant_id_scope_idx
+            ON entries(tenant_id, scope);
+`);
+}
+
+export async function down(knex: Knex): Promise<void> {
+    return knex.raw(`DROP INDEX IF EXISTS entries_tenant_id_scope_idx;`);
```

Let's not use `IF EXISTS` in the down transaction

---

The reason is that it might mask an error?

---

removed)